libs/expo/betterangels/src/lib/screens/AddEditClient/AddEditClient.graphql

```diff
@@ -23,12 +23,14 @@ query GetClientProfile($id: ID!) {
       dateOfBirth
       gender
       genderOther
+      displayGender
```

Comment:
Hey @Davit-BetterAngels 

I saw @johnr54321 's comment about the link for non-existed Current Manager. Just removed the link.
<img width="360" src="https://github.com/user-attachments/assets/f1743ea7-e296-42e7-90f3-3c93fbf4ee09" />

Reply:
@johnr54321 , it wasn't written in the AC. 
We need to create a ticket for this part.